### ISSUE TITLE
Failed to Generate

### ISSUE THREAD

**Issue by jgrammen-agilitypr** [bodgit/puppet-postfix#24]
```
Error: /Stage[main]/Postfix::Config/Resources[postfix_master]: Failed to generate additional resources using 'generate': Augeas didn't load /etc/postfix/master.cf with Postfix_Master.lns from /opt/puppetlabs/puppet/cache/lib/augeas/lenses: Iterated lens matched less than it should (line:67, character:14)
```
installed the following dependancies
```
puppetlabs/stdlib 4.25.1
herculesteam/augeasproviders_core 2.6
camptocamp/augeas 1.9.0
```

Running puppet 6

**Comment by bodgit** [comment]
What's installed that lens? That looks like it has been pluginsync'd from another module? If you delete it does it get reinstalled? My module uses `camptocamp/augeas` to install the updated lens which AFAIK installs it to `/opt/puppetlabs/puppet/share/augeas/lenses`.

It looks like the bug you're hitting is the one I fixed in #16 which is due to the last released version of Augeas shipping a lens that doesn't handle Postfix services that are of type `unix-dgram`.

**Comment by jgrammen-agilitypr** [comment]
I apologize after having running puppet, it appears the error has gone away. Probably an installation issue. It may have been because i didn't have the dependencies installed.
